website/build.sh

```diff
@@ -18,12 +18,15 @@ fi
 
 cp -r prep primers projects website/content/
 
-mkdir -p website/content/about
 cp CONTRIBUTING.md website/content/about/contributing.md
 
 for file in $(find website/content/projects -name README.md); do
   mv "${file}" "${file%README.md}index.md"
 done
 
+for file in $(find website/content/projects -maxdepth 1 -mindepth 1 -name index.md ) ; do
+  mv "${file}" "${file%index.md}_index.md"
+done
```

Comment:
I think this is what you're achieving? This seems clearer, and avoids a chain of double-renames?

```suggestion
mv website/content/projects/README.md website/content/projects/_index.md

for file in $(find website/content/projects -name README.md); do
  mv "${file}" "${file%README.md}index.md"
done
```